HaxeManual/02-types.tex

```diff
@@ -597,7 +597,7 @@ \subsection{Extensions}
 
 \since{4.0.0}
 
-Haxe 4 introduces an alternative notation for extension, separating each extended structure with a \expr{\&} symbol.
+Haxe 4 introduces an alternative notation for an extension, separating each extended structure with a \expr{\&} symbol.
```

Comment:
Unsure about this. If the original sounds too wrong I would go for `alternative notation for extensions`

Reply:
I think the original sounds just fine.

Reply:
"for extensions" sounds better